eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -78,6 +81,14 @@ public String hash() {
         return result;
     }
 
+    /**
+     * Short version of hash.
+     * @return SHA of commit
+     */
+    public String shortHash() {
```

Comment:
@mximp why not just `short()`?

Reply:
@yegor256 `short` is reserved identifier in Java

Reply:
@mximp maybe `small` then? compound names are bad ideas in general: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html